### DISCUSSION TITLE
How can I use `vcode` types from the debug console?

### DISCUSSION THREAD

**Discussion by tinder-tannerbennett** [microsoft/vscode-discussions#182] [Extension Development QnA] [answered]
<img width="564" alt="Screen Shot 2022-10-16 at 7 10 51 PM" src="https://user-images.githubusercontent.com/89099176/196074423-23aa625d-7c35-4fae-8576-e9b1a9b87dcd.png">

I'm not able to test things like `Uri.joinPath()` in the console; `Uri is not defined`. I also can't `require('vscode')` or `import` it from the console.

**Comment by carlocardella** [comment, marked as answer]
Use `vscode_1`, all built-in types are available there: 

![image](https://user-images.githubusercontent.com/5784415/198464203-01c3b1f1-8e76-4f7f-ad0c-57f450cc8fb0.png)

If you group your code in modules, those functions and types are likewise available with a similar naming convention (`modulename_1`) and you can use those in the debug console as well:

![image](https://user-images.githubusercontent.com/5784415/198467127-4dc87434-ea55-42cf-91d5-b19f2e5c25e8.png)

**Reply by tinder-tannerbennett** [reply]
I know it's been a year, but I forgot to say thanks :)
